perllib/FixMyStreet/App/Engine.pm

```diff
@@ -0,0 +1,68 @@
+package FixMyStreet::App::Engine;
+
+use Moose;
+extends 'Catalyst::Engine';
```

Comment:
It looks like we already have our own version of `Catalyst::Engine` so why not just change `finalize_cookies` in that?

Reply:
The Catalyst/Engine.pm is a copy of Catalyst's without changes (just a more recent version), so I thought it was clearer to keep that and subclass it properly.